vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ABIViewManager.h"
+
+#include <ReactUWP/Utils/ValueUtils.h>
+#include "ReactSupport.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+ABIViewManager::ABIViewManager(
+    const std::shared_ptr<react::uwp::IReactInstance> &reactInstance,
+    const winrt::Microsoft::ReactNative::Bridge::IViewManager &viewManager)
+    : Super(reactInstance) {
+  if (nullptr == viewManager) {
+    throw winrt::hresult_null_argument(L"viewManager");
+  }
+  m_viewManager = viewManager;
```

Comment:
>m_viewManager [](start = 2, length = 13)

In C++ it is always better to initialize fields in the initialization list to avoid double field initialization.